ern-local-cli/src/lib/utils.js

```diff
@@ -635,6 +636,62 @@ async function performPkgNameConflictCheck (name: string) : Promise<boolean> {
   return true // If package name doesn't exist continue with command execution
 }
 
+function checkIfModuleSuffixExists (moduleName: string, moduleType: string): boolean {
```

Comment:
This function does not check if the module suffix exists. It checks if the module name contains the suffix, but can be in any location of the module name. 
Should fix the function so that it does what it says it does ;)
You can achieve it with some string manipulation, but probably easier through a regex.

Reply:
@belemaire The logic already handles logic, i'll rename the function instead. Thanks